test/controllers/forms/EducationQuestionnaireFormSpec.scala

```diff
@@ -99,10 +99,10 @@ class EducationQuestionnaireFormSpec extends BaseSpec {
     }
 
     "transform form when form is full valid (has degree and lived in uk) to a question list" in new Fixture {
-      val questionList = FullValidForm.exchange.questions
+      val questionList = FullValidForm.exchange().questions
       questionList.size mustBe 8
-      questionList(0).answer.answer mustBe Some("Yes")
-      questionList(0).answer.unknown mustBe None
+      questionList.head.answer.answer mustBe Some("Yes")
```

Comment:
Not sure that the readability is increased by using head where we then subsequently refer to other indices